### PR TITLE
Spinner with events and catch error in initialize

### PR DESCRIPTION
**Pull request by haroldorodsilva**
Option to get the events on create in events.

### CODE REVIEW

**Comment by haroldorodsilva** [comment]
efb0894 is to prevent chrome to continue opened. Some times I get the error and don't get the timeout to finish the aplication. The solution that I find out is to kill the page if get anything of error on create.

**Comment by smashah** [comment]
Thanks for this. I'll review in the upcoming week

File: src/controllers/initializer.ts [hunk 1]
```diff
@@ -1,22 +1,33 @@
-import ora from 'ora';
 import { Whatsapp } from '../api/whatsapp';
 import * as path from 'path';
 import { isAuthenticated, isInsideChat, retrieveQR, randomMouseMovements } from './auth';
 import { initWhatsapp, injectApi } from './browser';
-const spinner = ora();
-let shouldLoop = true;
+import { EventEmitter2 } from 'eventemitter2';
+import * as spinner from './step';
```
[the remark this answers is not in the document]
Comment (haroldorodsilva) [reply src/controllers/initializer.ts]: Great!

**Comment by smashah** [comment]
@haroldorodsilva check out latest version 1.8.3 It has this functionality built-in now.

**Comment by haroldorodsilva** [comment]
@smashah and about the try catch to kill the puppeter ?

**Comment by smashah** [comment]
@haroldorodsilva implemented. That will be live on next release but less required since TOS_BLOCK is fixed. Thanks.

**Pull request closed without merging**